tests/nnapi/specs/V1_1/transpose_2D_nnfw.mod.py

```diff
@@ -1,13 +1,15 @@
 model = Model()
 i1 = Input("input", "TENSOR_FLOAT32", "{3,4}")
-perms = Parameter("perms", "TENSOR_INT32", "{2}", [1, 0])
+perms = Input("perms", "TENSOR_INT32", "{2}")
```

Comment:
Please do not modify this test file.

Reply:
Reverted the changes, with reasoning given in Issue #1875 . Thank you for the discussion.